src/main/java/uk/gov/companieshouse/pscstatementdataapi/config/SimpleAuthenticationInterceptor.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.companieshouse.pscstatementdataapi.config;
```

Comment:
I don't think this class should sit inside config. Could it be put inside a new package for auth/authorisation??